### DISCUSSION TITLE
INAV F4 Support

### DISCUSSION THREAD

**Discussion by dodo124** [iNavFlight/inav#8328] [Q&A] [answered]
Hello and thank you for this great pice of software :-)

I like it very much and use it for a long time.

And now I found an INAV F4 Flight controller (laying around) which has to be used ;-)

While it is found via CLI  in iNav5 it needs to be updated to do more.

Unfortunately I don´t find a appropriate firmware in the list.

The FC is this one: [https://de.aliexpress.com/i/32952753289.html](url)

Any recommendatons which firmware to use?

BTW: where can I download iNav 4.0? did not find it on github all references point to iNav5

Thanks Dom

**Comment by DzikuVx** [comment, marked as answer]
INAV has no idea what it is. INAV project was never consulted about this board. All questions whould be directed to the manufacturer of this hardware

**Reply by dodo124** [reply]
Ok that the explanation. Can you tell me where to find the inav 4.0 because it´s supposed to work with that.
Thanks for your help

Greeting Dom

**Reply by erstec** [reply]
Probably here https://github.com/iNavFlight/inav-configurator/releases/tag/4.0.0

**Reply by dodo124** [reply]
Thanks. I found some info in the cli about the chip. It´s compatible with FF-F35 Lightning.
Thanks Dom
